src/App.svelte

```diff
@@ -6,7 +6,7 @@
 	const routes = {
 	  '/': Home,
 	  '/about': About,
-	  '/question': Question,
+	  '/questions': Question,
```

Comment:
It seems you've put all the Questions page stuff on the Question page. Could you create a new component called "Questions" and not overide the Question route? If you override it, its could to add some merge conflicts for Adi.

Reply:
Should I just rename the current "Question" page to "Questions and make a new "Question" page?

Reply:
I would recommend having a new "Questions" page component in 

https://github.com/CarletonComputerScienceSociety/code-project/tree/questions_page/src/pages

The code you currently have in the "Question" page component should live there instead to avoid conflicts with Adi's changes